sway-lib-std/src/block.sw

```diff
@@ -19,11 +36,57 @@ pub fn height() -> u32 {
 }
 
 /// Get the TAI64 timestamp of the current block.
+///
+/// # Additional Information
+///
+/// The TAI64 timestamp begins at 2^62 seconds before 1970, and ends at 2^62 seconds after 1970,
+/// with a TAI second defined as the duration of 9192631770 periods of the radiation corresponding
+/// to the transition between the two hyperfine levels of the ground state of the cesium atom.
```

Comment:
woah

Reply:
is it fine?

Reply:
I just had no idea that this is how time was defined.

I think including the timestamp spec is useful, so mentioning the TAI64 spec, but instead of writing out the TAI64 definition perhaps including an external link to where TAI64 is defined would be better.

Good thinking about including this!

Reply:
I could not find any place to link which had a simple definition, so i tried to describe it shortly myself